src/pshtt/pshtt.py

```diff
@@ -1594,21 +1592,33 @@ def load_preload_list():
     return fully_preloaded
 
 
-# Returns an instantiated PublicSuffixList object, and the
-# list of lines read from the file.
-def load_suffix_list():
+# Returns an instantiated PublicSuffixList object.
+def load_suffix_list(cache_suffix_list=None, update_list=False):
     """Download and load the public suffix list."""
-    # File does not exist, download current list and cache it at given location.
-    utils.debug("Downloading the Public Suffix List...", divider=True)
-    try:
-        cache_file = fetch()
-    except URLError as err:
-        logging.exception("Unable to download the Public Suffix List...")
-        utils.debug(err)
-        return []
-    content = cache_file.readlines()
-    suffixes = PublicSuffixList(content)
-    return suffixes, content
+    if update_list:
+        utils.debug("Downloading the Public Suffix List...", divider=True)
+        try:
+            # Update the local copy
+            if cache_suffix_list:
+                updatePSL(cache_suffix_list)
+            # Update the built-in copy
+            else:
+                updatePSL()
+        except Exception as err:
+            logging.exception("Unable to download the Public Suffix List...")
+            utils.debug(err)
+            return None
+
+    # Use the local copy
+    if cache_suffix_list:
+        utils.debug("Using cached Chrome preload list.", divider=True)
```

Comment:
Is this debug message correct?  Or should it be this:
```suggestion
        utils.debug("Using cached Public Suffix List.", divider=True)
```